pkg/dockerfile/parse.go

```diff
@@ -73,6 +73,24 @@ func (d *Dockerfile) FindBaseImage(buildArgs map[string]string, target string) s
 	return ""
 }
 
+// BuildContextFiles traverses a build stage and returns a list of any file path that would affect the build context
+func (d *Dockerfile) BuildContextFiles() (files []string) {
```

Comment:
some unit tests for this would be great, just to be safe

Reply:
good point!